frontend/app/views/spree/products/_filters_mobile.html.erb

```diff
@@ -51,6 +51,10 @@
             </div>
           </div>
         </div>
+
+        <% additional_mobile_filters_partials.each do |partial| %>
+          <%= render partial, permitted_params: permitted_params %>
+        <% end %>
```

Comment:
```suggestion
        <% additional_mobile_filters_partials.each do |partial| %>
          <%= render "mobile_#{partial}", permitted_params: permitted_params %>
        <% end %>
```
something like that should work and there would be less configuration.
Actually making single partial for all filters of the mobile/desktop type seems possible.